PyRDF/RDataFrame.py

```diff
@@ -33,9 +33,10 @@ def __init__(self, *args):
         try:
             ROOT.ROOT.RDataFrame(*args) # Check if the args are correct
         except TypeError as e:
-            raise RDataFrameException(e, "Error creating the RDataFrame !") from None
-            # 'from None' is to supress the traceback of
-            # the redundant error 'e'
+            rdf_exception = RDataFrameException(e, "Error creating the RDataFrame !")
```

Comment:
I understand this will still show the error message coming from C++, right? The one saying `None of the overloads succeded`.
Also, could we add a unit test that just checks the construction of the PyRDF? With the right arguments and with wrong arguments (assert exception). With supported and unsupported operations.

Reply:
Indeed, that test could also ensure that no other exception is raised, other than `RDataFrameException`.

Reply:
Sure @etejedor and @JavierCVilla . I'll write a test for this 👍